static/css/components/readerStats.less

```diff
@@ -8,14 +8,30 @@
   font-size: 0.9em;
   li {
     list-style-type: none;
-    display: inline;
+    display: inline-block;
     &:not(:last-child):after {
       content:'·';
       margin: 0 4px;
     }
   }
+  
   &__star {
-    color: @orange;
-    font-size: 1.3em;
+    --star-size: 25px;
```

Comment:
Does LESS transpile these CSS variables ? If not, what is the impact of this on older browsers where CSS3 is not supported?

Reply:
I have removed css variables and changed --percent to @percent which uses --rating variable from the HTML. So overall it transpile to:
![image](https://user-images.githubusercontent.com/64412143/108866218-003a9a80-761a-11eb-9fe8-17fe393b38db.png)
![image](https://user-images.githubusercontent.com/64412143/108866267-0c265c80-761a-11eb-9b09-b6a7592c4785.png)

Reply:
Am still seing a calc in that screenshot..

Reply:
@jdlrobson I have removed the calc by calculating the percentage directly in HTML file. But I am not getting How to remove the var. Can you please guide me?
![image](https://user-images.githubusercontent.com/64412143/110230395-9766f280-7f36-11eb-8d49-5eb6056a947a.png)

Reply:
This approach is not going to work for older browsers. You can't communicate LESS variables from HTML.

I don't think we need this level of precision. Rounding to nearest 0.5 would be fine.

I would recommend using the original approach but adding a new class readers-stats__star--half to cover the hal;f case.

Reply:
I have made the changes. Please take another look